booking.html

```diff
@@ -6,6 +6,7 @@
     <title>Booking</title>
     <link rel="stylesheet" href="static/css/main.css" />
     <script type="module" src="src/JS/booking.js" defer></script>
+    <script type="module" src="src/JS/booking.js" defer></script>
```

Comment:
Should change one of the script tags to link "main.js". Right now the hamburger menu doesnt work in booking.html

Reply:
Done!